server/src/main/java/org/opensearch/telemetry/TelemetrySettings.java

```diff
@@ -64,20 +67,59 @@ public class TelemetrySettings {
         Setting.Property.Final
     );
 
+    /**
+     * Probability of action based sampler
+     */
+    public static final Setting<Double> TRACER_SAMPLER_ACTION_PROBABILITY = Setting.doubleSetting(
```

Comment:
Please move those to `OTelTelemetrySettings`

Reply:
@Gaganjuneja Do need to move this also ? Or just `TRACER_SPAN_SAMPLER_CLASSES`

Reply:
`TRACER_SAMPLER_ACTION_PROBABILITY` would only be provided by OtelPlugin and should be moved there

Reply:
@devagarwal1803 we use cluster settings for settings that needs to be respected / supported by any component out there. The `TRACER_SAMPLER_PROBABILITY` (added before) make sense - we want any telemetry to support probability sampling. The action sampler is purely specific to Otel plugin at the moment and should be moved there. It is fine to keep this setting static for now:
a) we should not and cannot make everything dynamic
b) this sampler would very luckily evolve to have action / probability pairs or be removed if not used

Also, it is not very useful to keep action static but probability dynamic.